### ISSUE TITLE
Manage viz R packages required (1)

### ISSUE THREAD

**Issue by jiwalker-usgs** [USGS-VIZLAB/vizlab#91]
The setup to get a viz to build requires all packages being in place.  There should be a section in viz.yaml or somewhere else reasonable to define R install details (CRAN, GRAN, github, ...).  Also, these need to be loaded so we can remove `library()` calls from viz scripts.

**Comment by eread-usgs** [comment]
For setup, need to figure out what is needed to install. If VizLab package is modified, then there's no way to indicate if github versus another version is needed. 

make as a section in the viz.yaml.

**Comment by eread-usgs** [comment]
1) Where do we define our dependencies?
2) How do we install?
3) How do we load?

**Comment by eread-usgs** [comment]
do '= version number' not '> version number '

**Comment by eread-usgs** [comment]
use rOxygen to create a namespace file?

**Comment by eread-usgs** [comment]
add libraries/packages section to viz skeleton

**Comment by aappling-usgs** [comment]
We have accomplished the first part of this task "section in viz.yaml or somewhere else reasonable to define R install details (CRAN, GRAN, github, ...)".

We've added `checkVizlabPackages()` to help with the second part ("How do we install?"), though it's not a complete solution because it doesn't do any automatic installation.

Remake (#246) would be one way to address the third part ("How do we load?"), though if we stick with make we'll need some other solution.
